File: manuals/lua/primetest.lua
function is_prime(x)
    if (x == 1 or x % 2 == 0 and x ~= 2) then
        return false
    else
        for i = 3, math.sqrt(x), 2 do
            if (x % i == 0) then
                return false
            end
        end
        return true
    end
end

for j = 1, 200 do
    if is_prime(j) then
        io.write(j .. " ")
    end
end
io.write("\n")

File: manuals/java/primetest.java
public class PrimeTest {

  public static void main(String[] args) {
    for (int i = 1; i < 200; i++) {
      if (is_prime(i)) {
        System.out.print(i + " ");
      }
    }
    System.out.println();
  }

  public static boolean is_prime(int x) {
    if (x == 1 || x % 2 == 0 && x != 2) {
      return false;
    } else {
      for (int i = 3; i <= Math.sqrt(x); i += 2) {
        if (x % i == 0) {
          return false;
        }
      }
      return true;
    }
  }
}
